data/categories.txt

```diff
@@ -2349,7 +2349,7 @@ nl:^Synagoge|tempel|gebedsplaats|gebedshuis
 fi:^Synagoga|uskonnollinen kohde
 fr:^Synagogue|lieu de culte|temple
 de:^Synagoge|Anbetungsstätte|Tempel
-hu:^Zsinagóga|templom
+hu:^Zsinagóga|imaház
```

Comment:
​No, a synagogoue is a jewish imaház (house of prayer). It is relativey common in Hungarian. Templom refers​ mostly to christian places, a synagogue is not a templom. (Historic reference: there was one jewish "templom", in Jerusalem, but it doesn't exist anymore since ~100 AD.)